projects/packages/search/src/dashboard/components/module-control/index.jsx

```diff
@@ -17,7 +17,7 @@ const SEARCH_DESCRIPTION = __(
 	'jetpack-search-pkg'
 );
 const INSTANT_SEARCH_DESCRIPTION = __(
-	'Instant search will allow your visitors to get search results as soon as they start typing. If deactivated, Jetpack Search will still optimize your search results but visitors will have to submit a search query before seeing any results.',
+	'Instant search uses a dynamic overlay for lightning-fast results, sorting, and filtering without reloading the page.',
```

Comment:
I could be wrong, but `lightning-fast results` doesn't sound quite right to me. How about `lightning-fast searching` or `lightning-fast search`?